modules/controlpanel.cpp

```diff
@@ -1130,7 +1132,7 @@ class CAdminMod : public CModule {
         CString sPass = sLine.Token(5);
 
         if (sServer.empty()) {
-            PutModule("Usage: DelServer <username> <network> <server>");
+            PutModule("Usage: DelServer <username> <network> <server> [port] [password]");
```

Comment:
Wouldn't it be enough to just change the `Usage` messages and to leave the rest of the code unchanged?
The rest of this commit actually loses functionality: You can no longer use `+6697` as a port number to specify SSL. Please look at what `CIRCNetwork::AddServer(const CString&)` actually does: https://github.com/znc/znc/blob/master/src/IRCNetwork.cpp#L1129-L1150